src/server/run_sched.c

```diff
@@ -253,94 +202,69 @@ find_sched_from_sock(int sock)
 	pbs_sched *psched;
 
 	for (psched = (pbs_sched*) GET_NEXT(svr_allscheds); psched; psched = (pbs_sched*) GET_NEXT(psched->sc_link)) {
-		if (psched->scheduler_sock == sock || psched->scheduler_sock2 == sock)
+		if (psched->sc_primary_conn == sock || psched->sc_secondary_conn == sock)
 			return psched;
 	}
 	return NULL;
 }
 
 /**
  * @brief
- * 		contact_sched - open connection to the scheduler and send it a command
- *		Jobid is passed if and only if the cmd is SCH_SCHEDULE_AJOB
+ * 		Connects to the Scheduler requested.
```

Comment:
This isn't correct, you aren't connecting to a scheduler.

Reply:
Remove this func as we now have easy & good attr setters